diff --git a/packages/vitest/src/node/cli/cli-config.ts b/packages/vitest/src/node/cli/cli-config.ts
--- a/packages/vitest/src/node/cli/cli-config.ts
+++ b/packages/vitest/src/node/cli/cli-config.ts
@@ -1,4 +1,3 @@
       ui: {
-        description:
-          'Show Vitest UI when running tests (default: `!process.env.CI`)',
+        description: 'Show Vitest UI when running tests (default: `!process.env.CI`)',
       },
